packages/pdfator-front/src/assets/pdfator-logo-right.svg

```diff
@@ -0,0 +1,80 @@
+<svg  viewBox="0 0 488 371" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
This is SVG is pretty heavy. You should use the same one for both left and right with a CSS rotation